MAUI/MauiAppBasic/AppCreationScripts/sample.json

```diff
@@ -0,0 +1,70 @@
+{
+  "Sample": {
+    "Title": "MAUI using MSAL.NET to acquire an access token and call Microsoft Graph API",
+    "Level": 200,
+    "Client": "MAUI (iOS, Android, UWP)",
+    "Service": "Microsoft Graph",
+    "RepositoryUrl": "active-directory-xamarin-native-v2",
+    "Endpoint": "AAD v2.0"
```

Comment:
>Endpoint [](http://example.com/codeflow?start=5&length=8)

Description: A .NET MAUI app using MSAL.NET to sign-in users and acquiring a token to call Microsoft Graph Api